### ISSUE TITLE
Ram Full After 4th plots with 64gb and rtx 3070

### ISSUE THREAD

**Issue by RealMacubex** [madMAx43v3r/chia-gigahorse#160]
My build is rtx3070 8gb  - amd 3950x - 64gb ram

i have try this config:

cuda_plot_k32 -n -20 -C 9 -t /mnt/ssd500gb/ -3 /mnt/ssd2TBRaid0/ -d @REMOTE_HOST -c <contract_address> -f <farmer_key>

After each plots p3 table 7, i see considerably increasing ram usage.

Is there any way to limit RAM usage?

does it make sense to use an ssd in raid 0 on -3 mode?

**Comment by zhurakovsky01** [comment]
No. Only GPU memory can be limited. Also, Max did not recommend creating C9 compression. It consumes 5 times more resources from C8. The best result would be: cuda_plot_k32 -n 1 **-S 2 -M 20** -C 9 -t /mnt/ssd500gb/ -3 /mnt/ssd2TBraid0/ -d @REMOTE_HOST -c <contract_address> -f <farmer_key>
